src/platform/packages/shared/kbn-mcp-dev-server/README.md

```diff
@@ -14,9 +14,9 @@ node scripts/mcp_dev
 
 This starts a [stdio](https://modelcontextprotocol.io/specification/2025-06-18/basic/transports#stdio) MCP Server.
 
-## Windsurf configuration example
+## MCP configuration example
 
-Add the following snippet to your `~/.codeium/windsurf/mcp_config.json` to let Windsurf automatically start/attach to the dev server (it assumes you use `nvm`):
+Add the following snippet to your settings for your AI Coding ide so it will automatically start/attach to the dev server (it assumes you use `nvm`):
```

Comment:
```suggestion
Add the following snippet to your settings for your AI Coding IDE so it will automatically start/attach to the dev server (it assumes you use `nvm`):
```